src/loops/simple_programs_with_repitition/easy/ReverseReverse.java

```diff
@@ -0,0 +1,23 @@
+package loops.simple_programs_with_repitition.easy;
+
+/*
+Write a program to reverse a number.
```

Comment:
This should have sample output in the question so they know how to code it properly, otherwise, they will just get an error saying their code does not match the right format.

Reply:
Your code is good, the document just needs to be changed

Reply:
And then the question comment in your code